### ISSUE TITLE
Apply *.sh configuration to shebang'ed files as well

### ISSUE THREAD

**Issue by mcandre** [editorconfig/editorconfig-vim#91]
Would be convenient if configuration styles for `*.sh` files were applied based on shebang as well, so that shell scripts without file extensions could be editorconfig'ed. Make sense?

**Comment by canaaerus** [comment]
Yes, please apply the options based on the `ftplugin`, so that all those `#!`-files (e.g. also python scripts) without extensions are correctly identified.

**Comment by cxw42** [comment]
Related to editorconfig/editorconfig#190 and editorconfig/editorconfig#239
